src/main/pages/workspace-admin/projects.adoc

```diff
@@ -34,28 +36,40 @@ Projects are always associated with a workspace and saved in a workspace configu
  ]
 ----
 
-Once a project is saved into a workspace config, the IDE will check if the project exists on a file system, and if not, import it using `source.location` URL. It happens at IDE initialization stage.
+Once a project is saved into a workspace configuration, the IDE will check if the project exists on a file system.  If the project does not exist on the file sytem, you can import it using the `source.location` URL. This happens during the IDE initialization stage.
 
-You can add Git projects, remotely hosted archives, your GitHub projects or project samples that Eclipse Che offers out of the box. Project import tools can be found both in User Dashboard (when creating a new workspace or editing an existing one) on in the IDE, Workspace menu.
+You can add the following projects:
 
-image::workspaces/projects.png[]
+. Git projects 
+. remotely hosted archives
+. GitHub projects 
```

Comment:
What's the diff. between Git and GitHub projects?

Reply:
I am not sure what that means here. but could it mean that Git projects are projects on your local machine and GitHub projects are projects stored on the cloud.

Reply:
Well, the idea here is that GitHub projects *are* Git projects. Is there some special support for GitHub versus any other Git hosting (GitLab, Bitbucket, a custom Git server, or local repositories)?